server/sql/subscriptions/getSubscriptionsByUserId.sql

```diff
@@ -0,0 +1,2 @@
+SELECT name, nickname, to_char(due_date, 'YYYY-MM-DD') AS due_date, reminder_days, amount, subscription_uuid FROM subscriptions 
+  WHERE(user_id = ${userId});
```

Comment:
for readability, this is how I usually format this type of query:

```suggestion
SELECT
  name,
  nickname,
  to_char(due_date, 'YYYY-MM-DD') AS due_date,
  reminder_days,
  amount,
  subscription_uuid
FROM subscriptions 
WHERE user_id = ${userId};
```